lib/AudioDecoder.js

```diff
@@ -20,7 +20,7 @@ class AudioDecoder extends Emitter
 		super();
 		
 		//Create decoder
-		this.decoder = new Native.AudioDecoderFacade();
+		this.decoder = new Native.AudioDecoderFacadeShared();
```

Comment:
an explicit `SharedPointer` call is needed so that TypeScript understands that `Native.AudioDecoderFacadeShared` also has the methods of `Native.AudioDecoderFacade`:
```suggestion
		this.decoder = SharedPointer(new Native.AudioDecoderFacadeShared());
```

Reply:
will fix soon